examples/jf-admin-app/linux/JFAManager.cpp

```diff
@@ -84,6 +98,46 @@ void JFAManager::HandleCommissioningCompleteEvent()
     }
 }
 
+bool JFAManager::IsDeviceJFAdmin()
+{
+    if (jfFabricIndex == kUndefinedFabricId)
```

Comment:
it is possible for a node to have multiple NOC entries in its fabric table.

Reply:
We're checking the CATs associated with a specific Fabric Index from the Fabric Table, so this shouldn't pose any issues.